modules/sdk-coin-trx/src/trx.ts

```diff
@@ -538,20 +548,88 @@ export class Trx extends BaseCoin {
       throw new Error('Invalid destination address!');
     }
 
+    let startIdx = params.startingScanIndex;
+    if (isUndefined(startIdx)) {
+      startIdx = 1;
+    } else if (!isInteger(startIdx) || startIdx < 0) {
```

Comment:
is it possible to have startIdx = 0 ?

Reply:
Receive address is seems to be starting with index 1. For base address we are already checking it seperately